src/test/java/org/springframework/data/redis/cache/DefaultRedisCacheWriterUnitTests.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017-2023 the original author or authors.
```

Comment:
Nit: Copyright starts at 2023 (inception of the file).

Reply:
Done